### DISCUSSION TITLE
友情链接问题

### DISCUSSION THREAD

**Discussion by Fgaoxing** [volantis-x/hexo-theme-volantis#734] [Q&A] [answered]
@Tikas 请问最新版相较于5.5和5.9友情链接页有啥变化吗？

**Comment by Tikas** [comment, marked as answer]
新版本移除了几个相关标签，你可以查看更新及文档里具体配置

[友链配置](https://vlts.cc/v5/page-settings/#%E5%8F%8B%E9%93%BE%E9%A1%B5%E9%9D%A2)
